avocado/core/safeloader.py

```diff
@@ -227,7 +247,32 @@ def get_methods_info(statement_body, class_tags):
     return methods_info
 
 
-def _examine_class(path, class_name):
+def _is_class_avocado_test(module, klass):
```

Comment:
Different than, say, `module_imported_as()`, which takes a `module` parameter that is the result for `ast.parse()`, this `_is_class_avocado_test()` utility function always operates on the basis of a module parameter that is an `AvocadoModule` instance.

So, it makes sense to have this as an utility *method*, of the `AvocadoModule` class instead.  Doing so also eliminates the confusion between what the `module` parameter here means.

Reply:
What I mean is best shown here:

https://github.com/avocado-framework/avocado/pull/3008/commits/9b6646c47db2b8e50e18bf4d60155cfea99b0eef